core/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -64,11 +65,27 @@
     public static final Setting<CircuitBreaker.Type> REQUEST_CIRCUIT_BREAKER_TYPE_SETTING =
         new Setting<>("indices.breaker.request.type", "memory", CircuitBreaker.Type::parseValue, Property.NodeScope);
 
-
+    public static final Setting<ByteSizeValue> TRANSPORT_CIRCUIT_BREAKER_LIMIT_SETTING =
+        Setting.byteSizeSetting("transport.breaker.request.limit",
+            new ByteSizeValue(128, ByteSizeUnit.MB), Property.Dynamic, Property.NodeScope);
```

Comment:
http.max_content_length is 100MB by default,  maybe we should allow at least two concurrent requests :-)